Splunk_TA_paloalto/lookups/pan_config_change.csv

```diff
@@ -0,0 +1,9 @@
+log_subtype,configuration_path,action,change_type,object,object_attrs,object_cateogry
```

Comment:
We noticed that in the merged code, this lookup is not included. This lookup will populate some of the CIM recommended fields for the Change Data Model. So, if the lookup is not included, the users won't be able to find the mentioned CIM fields for these events.

Reply:
The `configuration_path` values in the lookup table are from the log sample which represents only a handful of the thousands (tens of thousands?) of possible paths, and most of these paths have customer names in them which are not known in advance. It would be impossible to enumerate these, but they are required as input for the lookup. So I removed this for now because it isn't really doing anything.

I would like to add this back so my suggestion is we discuss what specific kinds of config changes we want to include CIM fields for (for example, administrator added) and create entries for those in the lookup table. Since this is additive and not a breaking change, we can do it at any time after the 7.0.0 release.